.dockerignore

```diff
@@ -4,4 +4,5 @@
 !**/*.mod
 !**/*.sum
 /bin
-/testbin
\ No newline at end of file
+/testbin
+db-operator
```

Comment:
This should create issues for the `dockerfile-ci`. there we copy direct the binary in the image, and ignore has a higher prio and should block the copy.

Reply:
Ouch, I forgot about that

Reply:
I think I can drop the `.dockerignore` commit, because the main issue was the`.gitignore`

Reply:
I've removed `.dockerignore`